content/about/use-case/_index.fr.md

```diff
@@ -5,24 +5,104 @@ description: Applications et axes directeurs
+ - application de différents types de marges sur les sillons, y compris une
+   marge économique, qui minimise l'énergie consommée
+
+Les perspectives futures comprennent notament :
+ - la détection automatique de conflits[^conflit]
+ - synthèse d'infrastructure macroscopique à partir d'infrastructure microscopique
+ - visualisation schématique du plan de transport
+ - module d'optimisation de la capacité en ligne, d'occupation de la voie en gare et de roulement matériel
+
+[^plan-exploitation]: Sous forme de plan d'exploitation
+[^schema-desserte]: Un schéma de desserte est une demande d'offre de transport, qui comprend:
+    - la représentation des trains demandés
+    - leur itinéraire
+    - les arrêts à desservir
+
+[^conflit]: Un conflit se produit lorsque la circulation d'un train en gêne un autre
+
+
+## Études d'exploitation
+
+Les études d'exploitation permettent d'évaluer la capacité de l'infrastructure à
+répondre à la demande et de la mettre en cohérence avec les besoins de transport.
+Elles permettent de prendre des décisions d'investissements d'infrastructure publique éclairées.
+
+OSRD permet de répondre à ce besoin en :
+ - fournissant une base d'infrastructure ferroviaires à l'échelle microscopique
+ - fournissant une base de donnée de matériel roulant
+ - donnant la possibilité de modifier l'infrastructure existante et ajouter des installations
+ - permettant de visualiser les grilles horaires
+ - permettant d'appliquer différents types de marges sur les sillons, dont une
+   répartition économique, qui minimise l'énergie consommée.
+ - fournissant un simulateur de signalisation
+
+Les pespectives futures comprennent notament :
+ - la détection de conflits[^conflit]
+ - l’évaluation de la robustesse d'une grille horaire via un module de simulation stochastique.
+ - calcul automatique de débit et capacité
+
+
+## Gestion d'infrastructure
+
+Le traitement d'informations liées à l'infrastructure ferroviaire étant un besoin
+commun à toutes les fonctionnalités d'OSRD, un module dédié permet de:
+ - visualiser sur une carte les différents types d'objets utilisables dans l'application
+ - éditer l'infrastructure ferroviaire
+ - rechercher des objets
+ - importer de nouvelles infrastructures[^railjson]
+
+![OSRD management](osrd_management.png)
+
+[^railjson]: RailJSON est le format de fichier d'infrastructure utilisé par OSRD.
+             Il est libre, documenté, conçu pour être agnostique au gestionnaire
+	     d'infrastructure, facile à générer et manipuler.
+
 
-<font color=#aa026d>
+## Sillons de dernière minute
 
-### Le projet OSRD
+Si une grande partie des sillons[^sillon] sont établis longtemps à
+l'avance, une proportion grandissante ne sont commandés que très peu
+de temps avant le départ souhaité du train, notament avec le développement
+du fret.
 
-</font>
+Il faut par conséquent être capable de répondre non seulement rapidement
+(moins de 3 minutes) à ces demandes, mais aussi correctement.
```

Comment:
This means: an automatic response must be possible…

Reply:
TODO: explicitly mention this is automated